packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/ModelStub.java

```diff
@@ -0,0 +1,19 @@
+package software.uncharted.terarium.hmiserver.models.dataservice;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+import java.io.Serializable;
+
+/**
+ * This is used to mirror model-creation and model-update
+**/
+@Data
+@Accessors(chain = true)
+public class ModelStub implements Serializable {
```

Comment:
From what I understand you do not need `Serializable`

Reply:
Can you point to a reference of not using serializable? 
Is this some Lombok magic because Java beans had always had to implement Serializable interface.

Reply:
From what I understand `Serializable` is necessary if we need for example to cache those object in Redis, etc.
But we do not need it yet.

Reply:
Correct, it's not strictly necessary at this point.